content/post/formatting-with-style.md

```diff
@@ -0,0 +1,256 @@
+---
```

Comment:
I like the blog post a lot, but I think it could use some more specific examples of how this actually works and what a given rule might look like.